linchpin/examples/example7/README.md

```diff
@@ -0,0 +1,97 @@
+Example: Adding a new kickstart test
+------------------------------------
+
+Run a *test run* with a single new test to be eventually added to `kickstart-test` repository.
+
+A small *target* will be provisioned and the test will be run possibly several times to make it right. At the end the *target* will be destroyed.
+
+#### Resources configuration:
+
+[PinFile](PinFile)
+
+contains these modifications of the [template pinfile](../../PinFile):
+
+* `count` - to use only single *master* *runner* (modified both in the topology and layout)
+* `flavor` - to use smaller instance as only one test is run
+
+#### Test configuration:
+
+[test-configuration.yml](test-configuration.yml)
+
+contains modifications to the [default configuration](../../../ansible/roles/kstest-master/defaults/main/test-configuration.yml)
+
+* `kstest_boot_iso_url` - `boot.iso` with installer to be tested
+* `kstest_url` - url of the installation repository to be used for the test (should correspond to the `kstest_boot_iso_url`)
+* `kstest_tests_to_run` - run only `new-test` test
+* `kstest_test_jobs` - run only one test in parallel on a *runner*
+* `kstest_git_repo` - the `kickstart-tests` repository with the `new-test` test
+* `kstest_git_version` - the repository branch with the `new-test` test
+
+
+#### Running the test(s)
+
+The script needs to be run from `kickstart-tests` repository.
+
+1) Provision the *target*.
+
+```
+./kstests-in-cloud.sh provision example7 --pinfile examples/example7/PinFile
+```
+
+2) Create and add the `new-test` test to a remote kickstart-test repository
+
+```
+git checkout -b new-test
+vim new-test.sh
+vim new-test.ks.in
+git add new-test.sh
+git add new-test.ks.in
+git commit
+git push origin new-test
```

Comment:
Please change the origin name here to something more specific. Origin is often used for the upstream.